applications/rag/notebook_starter.py

```diff
@@ -0,0 +1,20 @@
+import requests
+import sys
+
+api_url = 'http://127.0.0.1:9443/hub/api'
+
+token = sys.argv[1].strip()
+print(token)
+
+r = requests.post(api_url + '/users/admin/server',
+    headers = {
+        'Authorization' : f'token {token}',
+    },
+    json={},
+)
+
+if not r.ok:
```

Comment:
Does `r.raise_for_status()` not check if the status code is 2XX?

Reply:
If so we can remove this extra check?

Reply:
Sorry I meant to keep the raise_for_status and remove the other check. Either way is probably fine